news/_posts/2017-11-21-eindejaarsactie.markdown

```diff
@@ -0,0 +1,19 @@
+---
+layout: post
+title:  "Eindejaarsactie"
+date:   2017-11-21 20:00:00
+description: Eindejaarsactie 2017.
+permalink: /news/2017-11-21-eindejaarsactie/
+---
+
+Naar aanleiding van de aankomende eindejaarsfeesten organiseert Basket Lummen, in samenwerking met Prik&Tik Drinkcenter Borgers en De kaarskesprocessie Leonidas Lummen, een wijn- en pralinesverkoop ten voordele van de club. 
```

Comment:
De feestdagen komen eraan! Basket Lummen, Prik & Tik Drinkcenter Borgers en De Kaarskesprocessie slaan de handen in elkaar. Vanaf nu zijn we niet alleen een prachtige basketclub, maar ook wijn- en pralineverkopers.